index.html

```diff
@@ -30,6 +30,148 @@ <h6><a href="#">LET’S CONNECT</a></h6>
                 </ul>
             </div>
         </div>
-    </div>   
+    </div> 
+    <main> 
+        <section class="Works_Section">
+            <div class="card card_1">
+                <img src="/images/Snapshot1.png" alt="Snapshoot_Portfolio_1">
```

Comment:
- [ ] Your image link is broken, kindly fix it by adding a `.` at the beginning.
- [ ] Kindly do the same for all the images of the project.